src/scripts/background.js

```diff
@@ -18,6 +19,22 @@ const shouldTriggerNotification = (state, seconds) => {
   );
 };
 
+function randomInt (min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function jitter () {
+  // Random amount between 0-5 seconds
+  return randomInt(0, 5);
+}
+
+function backoff (retryInterval, retryCount) {
+  return Math.min(
+    (retryInterval * Math.pow(2, retryCount)) + jitter(),
```

Comment:
On 1st retry (so we've failed to connect once), retryCount is 0, correct? That's how I think the code works.. and in that case this is correct. Just want to confirm with you.

Reply:
Yep so 5 + 0 + rand(0, 5)